FreeRTOS/FreeRTOS
# Update Backward Compatibility Flag

Description
-----------
Update the backward compatibility flag for IPv4 only build to ipconfigIPv4_BACKWARD_COMPATIBLE.

Test Steps
-----------
Demo runs on Jenkins.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
